src/mmw/js/src/app.js

```diff
@@ -122,18 +122,41 @@ function RestAPI() {
 }
 
 function initializeShutterbug() {
+    var googleTileLayerSelector = '#map > .leaflet-google-layer > div > div > div:nth-child(1) > div:nth-child(1)';
+
     $(window)
         .on('shutterbug-saycheese', function() {
             // Set fixed width before screenshot to constrain width to viewport
             $('#model-output-wrapper, body > .map-container').css({
                 'width': window.innerWidth
             });
+            // Convert Google Maps CSS Transforms to Left / Right
+            var $googleTileLayer = $(googleTileLayerSelector),
```

Comment:
The `googleTileLayerSelector` is tuned to select only Google tile layers. If they are not visible, it returns an empty array, and none of the following operations will have any effect.